src/content/tools/_index.md

```diff
@@ -51,4 +52,4 @@ Valid OSCAL is open-ended in application. Some of the tools described provide va
 | [OSCAL Deep Diff](https://github.com/usnistgov/oscal-deep-diff) | A Typescript-based CLI application and library that produces machine readable and human-consumable comparisons of JSON OSCAL artifacts. | open source |
 | [OSCAL Java Library](https://github.com/usnistgov/liboscal-java) | A Java-based programming API for reading and writing content conformant to the OSCAL XML, JSON, and YAML based models. | open source |
 | [XSLT Tooling](https://github.com/usnistgov/oscal-tools/tree/main/xslt) | A variety of Extensible Stylesheet Language (XSL) Transformations (XSLT), Cascading Style Sheets (CSS), and related utilities for authoring, converting, and publishing OSCAL content in various forms. | open source |
-| [OSCAL Java CLI](https://github.com/usnistgov/oscal-cli) | An OSCAL Java command-line tool able to validate OSCAL content and metaschema for well formatting, to convert such content between the OSCAL XML, JSON and YAML formats, to resolve OSCAL profiles, and to generateXML and JSON schemas from a Metachema model definition. | open source |
\ No newline at end of file
+| [OSCAL Java CLI](https://github.com/usnistgov/oscal-cli) | An OSCAL Java command-line tool able to validate OSCAL content and metaschema for well formatting, to convert such content between the OSCAL XML, JSON and YAML formats, to resolve OSCAL profiles, and to generateXML and JSON schemas from a Metaschema model definition. | open source |
```

Comment:
Minor, non-blocking: we now should call the Metaschema modules to be more clear.

```suggestion
| [OSCAL Java CLI](https://github.com/usnistgov/oscal-cli) | An OSCAL Java command-line tool able to validate OSCAL content and metaschema for well formatting, to convert such content between the OSCAL XML, JSON and YAML formats, to resolve OSCAL profiles, and to generateXML and JSON schemas from Metaschema module(s). | open source |
```